test/e2e/snaps/test-snap-siginsights.spec.js

```diff
@@ -314,23 +234,13 @@ describe('Test Snap Signature Insights', function () {
         await switchToNotificationWindow(driver, 4);
 
         // wait for and click sign
-        await driver.waitForSelector({
-          text: 'Sign',
-          tag: 'button',
-        });
-        await driver.clickElement({
-          text: 'Sign',
-          tag: 'button',
+        await clickSignOnSignatureConfirmation({
+          driver,
+          snapSigInsights: true,
+          locatorID: '#ethSign',
         });
 
         // wait for and click signature warning sign button
-        await driver.waitForSelector(
-          '[data-testid="signature-warning-sign-button"]',
-        );
-        await driver.clickElement(
-          '[data-testid="signature-warning-sign-button"]',
-        );
-
```

Comment:
we don't need to confirm the warning since now this is handled inside the `clickSignOnSignatureConfirmation` method